### PR TITLE
Adopt the OpenJSF Code of Conduct

### PR DESCRIPTION
**Pull request by craigtaub**
### Description of the Change

First task for https://github.com/mochajs/admin/issues/7. Just a blanket copy-paste from [here](https://github.com/openjs-foundation/cross-project-council/blob/master/CODE_OF_CONDUCT.md).
Let me know if you think there is a nicer format or way to do this.

### CODE REVIEW

**Comment by boneskull** [comment]
@munter @outsideris @juergba Please take a look at this

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -56,3 +56,39 @@ Original text courtesy of the Speak Up! project and Django Project.
...
+- [english](https://www.contributor-covenant.org/version/1/4/code-of-conduct)
+- [translations](https://www.contributor-covenant.org/translations)
+
+Refer to the section on reporting and escalation in this document for the specific emails that can be used to report and escalate issues.
+
+## Reporting
+
+### Project Spaces
+
+For reporting issues in spaces related to a member project please use the email provided by the project for reporting. Projects handle CoC issues related to the spaces that they maintain. Projects maintainers commit to:
```
Comment (outsideris) [inline .github/CODE_OF_CONDUCT.md]: `please use the email provided by the project for reporting`

Have we the email?

Comment (craigtaub) [reply .github/CODE_OF_CONDUCT.md]: Good catch. 
Sounds like pointing to Gitter is not acceptable. Altho seems overkill to introduce an email just for this, but if its part of the CoC 🤷‍♂ .

@boneskull @Munter @juergba any thoughts?

File: .github/CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -56,3 +56,39 @@ Original text courtesy of the Speak Up! project and Django Project.
 ## QUESTIONS?
 
 If you have questions, please see the FAQ. If that doesn’t answer your questions, feel free to email conduct@js.foundation.
+
```
Comment (juergba) [inline .github/CODE_OF_CONDUCT.md]: `conduct@js.foundation` probably is outdated, should be something with openjs.

File: .github/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (juergba) [reply .github/CODE_OF_CONDUCT.md]: If I call a hotline to complain about a service and I never get an answer, that's just a pure provocation. Better no hotline at all, than a hotline that doesn't really care.
So an email without defining a responsible maintainer: IMO no.
Do we have resources for this: IMO no.

File: .github/CODE_OF_CONDUCT.md (hunk 2 above)
Comment (craigtaub) [reply .github/CODE_OF_CONDUCT.md]: I'll update to `report@lists.openjsf.org`

File: .github/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (outsideris) [reply .github/CODE_OF_CONDUCT.md]: I agree with no hotline. 
So, we will remove this? And can we remove this because it is OpenJS template?

Comment (craigtaub) [reply .github/CODE_OF_CONDUCT.md]: I didnt think we should remove it for that reason. I feel as a whole we do adhere to the `Projects maintainers commit to` points.

**Comment by craigtaub** [comment]
@Munter think your the last one left to look at this. Pls have a look when u get a mo. Thanks

File: .github/CODE_OF_CONDUCT.md (hunk 1 above)
Comment (boneskull) [reply .github/CODE_OF_CONDUCT.md]: I will be the point of contact for this for now

**Comment by boneskull** [comment]
It’s been sitting long enough I think it’s ok to merge.

**Pull request merged**